Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 878751a31888314a83c97035fde266c7c8a495a3)
+++ /dev/null	(revision 878751a31888314a83c97035fde266c7c8a495a3)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: ../Server/ServerChoDienTu/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.application.name=ServerChoDienTu\r\n\r\n\r\nspring.datasource.url=jdbc:mysql://localhost:3306/chodientu_db?createDatabaseIfNotExist=true&useSSL=false&serverTimezone=UTC\r\nspring.datasource.username=root\r\nspring.datasource.password=\r\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\r\n\r\nspring.jpa.show-sql=true\r\nspring.jpa.hibernate.ddl-auto=validate\r\n\r\nspring.mail.host=smtp.gmail.com\r\nspring.mail.port=587\r\nspring.mail.username=ktm6789.it@gmail.com\r\nspring.mail.password=rcvm feof zukx ahmd\r\nspring.mail.properties.mail.smtp.auth=true\r\nspring.mail.properties.mail.smtp.starttls.enable=true\r\n\r\njwt.secret=123456789012345678901234567890kha\r\njwt.expiration=3600000\r\njwt.refreshExperation=604800000\r\n\r\nspring.servlet.multipart.max-file-size=5MB\r\nspring.servlet.multipart.max-request-size=20MB\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/../Server/ServerChoDienTu/src/main/resources/application.properties b/../Server/ServerChoDienTu/src/main/resources/application.properties
--- a/../Server/ServerChoDienTu/src/main/resources/application.properties	(revision 878751a31888314a83c97035fde266c7c8a495a3)
+++ b/../Server/ServerChoDienTu/src/main/resources/application.properties	(date 1769331327075)
@@ -1,7 +1,7 @@
 spring.application.name=ServerChoDienTu
 
 
-spring.datasource.url=jdbc:mysql://localhost:3306/chodientu_db?createDatabaseIfNotExist=true&useSSL=false&serverTimezone=UTC
+spring.datasource.url=jdbc:mysql://localhost:3306/test_db?createDatabaseIfNotExist=true&useSSL=false&serverTimezone=UTC
 spring.datasource.username=root
 spring.datasource.password=
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
